tgui/packages/tgui/interfaces/Orbit.tsx

```diff
@@ -1,33 +1,33 @@
 import { useBackend, useLocalState } from '../backend';
 import { filter, sortBy } from 'common/collections';
 import { capitalizeFirst, multiline } from 'common/string';
-import { Button, Collapsible, Icon, Input, Section, Stack } from '../components';
+import { Box, Button, Collapsible, Icon, Input, LabeledList, Section, Stack } from '../components';
 import { Window } from '../layouts';
 import { flow } from 'common/fp';
 
-type AntagGroup = [string, Observable[]];
+type AntagGroup = [string, Antags];
+
+type Antags = Array<Observable & { antag: string }>;
 
 type Data = {
-  alive: Observable[];
-  antagonists: Observable[];
-  dead: Observable[];
-  ghosts: Observable[];
-  misc: Observable[];
-  npcs: Observable[];
+  alive: POIs;
```

Comment:
can we not do this

Reply:
POI is a bad technical term and Observable is what it literally is

Reply:
Sorry I am trying to get into the habit of using the array<> syntax and it looked ugly